### DISCUSSION TITLE
Home page animation violates WCAG 2.0 A 2.2.2 Pause, stop, hide

### DISCUSSION THREAD

**Discussion by CharlesBelov** [community/community#17228] [Accessibility] [answered]
The [GitHub home page](https://github.com/) animation for not-logged-in users lasts longer than 5 seconds. Under [WCAG 2.0 A 2.2.2 Pause, stop, hide](https://www.w3.org/TR/WCAG20/#time-limits), a visible pause button is required.

**Comment by benjiallen** [comment]
@CharlesBelov i assume this is the same/similar issue documented in https://github.com/github-community/community/discussions/17230. Is that correct?

**Comment by CharlesBelov** [comment]
It's related. #17230 (respecting prefers-reduced-motion to automatically not serve any animation) is polite behavior which is not mandated by WCAG. #17228 (allow pausing any animation longer than 5 seconds) is mandated by WCAG 2.0 at the A level. They are complementary rather than duplicate or mutually exclusive.

I suppose I could have filed them as a single issue. I tend to treat technically different issues discrete from one another. That said, one could choose to consider them together.

**Comment by cehfisher** [comment, marked as answer]
Hello @CharlesBelov - thank you for being a part of our community and taking the time to submit feedback. I'm happy to report that a pause/play button has been added to the globe animation for the logged-out GitHub homepage experience.

I am including two screenshots, the first image shows the default state with a red arrow pointing to the "pause" button. The second image shows the toggled state with a red arrow pointing to the "play" button. You will see this button when your OS `prefers-reduced-motion` option is not set to true/toggled on.

Note: there is work still being done on the other issue you created called [GitHub home page does not respect prefers-reduced-motion](https://github.com/orgs/community/discussions/17230). We hope to have an update for that issue soon.

Image 1:
<img width="1043" alt="Screenshot that shows the default state with a red arrow pointing to the pause button." src="https://github.com/user-attachments/assets/c18eeec1-1b07-480f-a27b-769d15c6c3bf">

Image 2:
<img width="1043" alt="Screenshot that shows the toggled state with a red arrow pointing to the play button." src="https://github.com/user-attachments/assets/c4a6d16f-6d75-4b62-9373-0ab62e1e3ad8">
